boards/nordic/nrf54h20dk/Kconfig.nrf54h20dk

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NRF54H20DK
+	select SOC_NRF54H20_CPUAPP if BOARD_NRF54H20DK_NRF54H20_CPUAPP
+	select SOC_NRF54H20_CPURAD if BOARD_NRF54H20DK_NRF54H20_CPURAD
+	select SOC_NRF54H20_CPUPPR if BOARD_NRF54H20DK_NRF54H20_CPUPPR || \
+				      BOARD_NRF54H20DK_NRF54H20_CPUPPR_RAM
```

Comment:
we should now dexault to CONFIG_XIP=n for PPR, and introduce the `/xip` variant. EngA was done the other way around because 28K of RAM + Zephyr defaults lead to an unusable platform.

Reply:
done